CHANGELOG.md

```diff
@@ -0,0 +1,190 @@
+
+### Repositories
+ * uploading a duplicate file fails with undefined local variable or method upload_href for #<Actions::Pulp3::Repository::CommitUpload:0x00000000134e3f00> ([#33748](https://projects.theforeman.org/issues/33748), [532ce8a0](https://github.com/Katello/katello.git/commit/532ce8a03879a771ad835c8fcfc2d60a4bc7abfe))
+ * b'Update recommended repos for Satellite & Tools from 6.9 to 6.10' ([#33627](https://projects.theforeman.org/issues/33627), [d921227a](https://github.com/Katello/katello.git/commit/d921227a96e1533f09e8a9150c80835fb8b0afcf))
+ * Generic repository types don't work properly with multiple generic types ([#33625](https://projects.theforeman.org/issues/33625), [6d0e46b1](https://github.com/Katello/katello.git/commit/6d0e46b1deecccf23feb34d4df0ad69343b9d161), [50642a73](https://github.com/Katello/katello.git/commit/50642a73afefeaba10db14cdb5fd4baa1c36844c))
+ * Selecting certain products in "Red Hat Repositories" page renders a Blank Page in Satellite 6.10 ([#33544](https://projects.theforeman.org/issues/33544), [deff2d5f](https://github.com/Katello/katello.git/commit/deff2d5f478f98b41eee62f2bbae3821e6141946))
+ * Selected yum metadata checksum type on is not reflected in repomd.xml on a repo creation ([#33495](https://projects.theforeman.org/issues/33495), [d2ea3134](https://github.com/Katello/katello.git/commit/d2ea31348c420314246e45b820cd88f040502120))
+ * Remove OSTree filter from Red Hat Repositories drop down list ([#33493](https://projects.theforeman.org/issues/33493), [13a09871](https://github.com/Katello/katello.git/commit/13a098718447816fd3b54b054a4caa206051f9ce))
+ * Total steps: 0/0 in sync status ([#33472](https://projects.theforeman.org/issues/33472), [d30d3bbc](https://github.com/Katello/katello.git/commit/d30d3bbca803c837ff516c3a79a31a1c35dcebff))
+ * Background download policy is still referenced in a number of areas ([#33468](https://projects.theforeman.org/issues/33468), [295c71a5](https://github.com/Katello/katello.git/commit/295c71a5c8a6fe7f216e7723ef0598afabcb0367))
+ * Add repositories node to generic content unit rabl ([#33454](https://projects.theforeman.org/issues/33454), [28be3d12](https://github.com/Katello/katello.git/commit/28be3d12f9a7fe67ca69d2218ab877f0ec315fd7))
+ * Drop rake task katello:refresh_sync_schedule ([#33450](https://projects.theforeman.org/issues/33450), [3166eb28](https://github.com/Katello/katello.git/commit/3166eb2805f4cda4c094299a6891af81e0fa4dd0))
+ * UI shows 0 packages\\errata\\package_groups after a bad sync followed by a successful sync for the same repo ([#33443](https://projects.theforeman.org/issues/33443), [9ff6d950](https://github.com/Katello/katello.git/commit/9ff6d950aec6f6ad84cfb70bb9b6340a806e62d6))
+ * Error in logs when scanning for repository: undefined method `resolve_substitutions' for nil:NilClass ([#33383](https://projects.theforeman.org/issues/33383), [8cb4654d](https://github.com/Katello/katello.git/commit/8cb4654dde46d5194b9947d399a841512d3223db))
```

Comment:
The redmine title breaks the Github markdown here. I'd change the Redmine title and fix it here as well

Reply:
@jeremylenz updated the redmine and also the issue title here.